Research: ci-regression-checks failure

## Research
2026-08-25 22:28:20 UTC

## [CI Log](https://github.com/InternationalColorConsortium/iccDEV/actions/runs/32905179737/job/97987577666#step:4:344)
```
[100%] Linking CXX executable iccApplyProfiles
gmake[3]: *** Deleting file 'Tools/IccApplyProfiles/iccApplyProfiles'
gmake[3]: *** wait: No child processes.  Stop.
gmake[3]: *** Waiting for unfinished jobs....
gmake[3]: *** wait: No child processes.  Stop.
gmake[2]: *** wait: No child processes.  Stop.
gmake[2]: *** Waiting for unfinished jobs....
gmake[2]: *** wait: No child processes.  Stop.
gmake[1]: *** wait: No child processes.  Stop.
gmake[1]: *** Waiting for unfinished jobs....
gmake[1]: *** wait: No child processes.  Stop.
gmake: *** [Makefile:592: iccApplyProfiles] Error 2
Error: Process completed with exit code 124.
```

--- a/.github/workflows/ci-iccdev-tool-tests.yml
+++ b/.github/workflows/ci-iccdev-tool-tests.yml
@@ -1115,7 +1115,7 @@ jobs:
             local label="$1"
             shift
             echo "[START] ${label}"
-            timeout 60s "$@"
+            timeout 180s "$@"
             echo "[PASS] ${label}"
           }
 

